addons/base_import/static/src/js/import_action.js

```diff
@@ -111,9 +109,22 @@ var DataImport = AbstractAction.extend(ControlPanelMixin, {
         action.display_name = _t('Import a File'); // Displayed in the breadcrumbs
         this.do_not_change_match = false;
     },
+    /**
+     * @override
+     */
+    willStart: function () {
+        var self = this;
+        var def = this._rpc({
```

Comment:
i think you can just return here, and not call this._super

Reply:
@ged-odoo Done.